### DISCUSSION TITLE
Underline After Selected Tab

### DISCUSSION THREAD

**Discussion by HeadMonitor** [AvaloniaUI/Avalonia#13401] [Q&A] [answered]
On TabControl there is always an underline under the selected item. I cannot find any information on how to disable it. How can this be achieved?

**Comment by thevortexcloud** [comment]
It will be defined in the theme. You can either change the theme or override it. I am guessing it's this thing in the theme:

https://github.com/AvaloniaUI/Avalonia/blob/fcea3453ff62225c4be56cf2569edffc72e7402e/src/Avalonia.Themes.Fluent/Controls/TabStripItem.xaml#L46C12-L52C49

But that's a guess, it might be something else. Tabs have a lot of associated controls.

**Comment by HeadMonitor** [comment]
Thanks for the reply. According to what you provided I tried doing:
```
<Style Selector="TabItem:PART_SelectedPipe">
        <Setter Property="Background" Value="Transparent" />
</Style>
```
But didn't seem to work. Maybe I understood wrong? Apologies for the rookie question I am new to Avalonia.

**Reply by thevortexcloud** [reply, marked as answer]
Selectors in Avalonia work kind of like CSS selectors. As such that wont work. As a general guide for how the themes work, the article for WPF on control templates is a decent place to start. That said there are differences between how Avalonia and WPF work, but it's mostly the same.

https://learn.microsoft.com/en-us/dotnet/desktop/wpf/controls/how-to-create-apply-template?view=netdesktop-7.0


You want to target PART_SelectedPipe specifically.  Which you can _probably_ do with

```xaml
<Style Selector="TabStripItem:selected /template/ Border#PART_SelectedPipe">
<Setter Property="IsVisible" Value="false"/>
</Style>
```

But I am not guaranteeing that will work as I am working from memory here.

https://docs.avaloniaui.net/docs/styling/selectors

**Comment by cuikp** [comment]
For me it didn't work with the TabStripItem selector, but rather:

```
<Style Selector="TabItem:selected /template/ Border#PART_SelectedPipe">
	<Setter Property="IsVisible" Value="false"/>
</Style>

```
